sklearn/metrics/pairwise.py

```diff
@@ -397,7 +413,7 @@ def pairwise_distances_argmin_min(X, Y, axis=1, metric="euclidean",
         Valid values for metric are:
 
         - from scikit-learn: ['cityblock', 'cosine', 'euclidean', 'l1', 'l2',
-          'manhattan']
+          'manhattan', 'gower']
```

Comment:
I think there is a similar listing that needs updating in `pairwise_distances`

Reply:
The only other listing is PAIRED_DISTANCES, it was requested to remove gower from this one.